.circleci/config.yml

```diff
@@ -347,13 +351,13 @@ workflows:
               only: master
       - license-check
       - ui
-      - runtime
-      - core:
+      - core
+      - runtime:
           requires:
-            - runtime
+            - core
```

Comment:
runtime does not require core (introduced only in hirams PR)

Reply:
check now